GenderPayGap.WebUI/Views/ActionHubOld/Index.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@{

```

Comment:
I think this file should have "layout" in it's name, rather than "index".
Index implies it's the homepage of this folder.

In the `EightWaysGuidance` folder, we've gone for a structure:
* Pages
  * SomePage.cshtml
  * AnotherPage.cshtml
* EightWaysGuidanceLayout.cshtml

What do you think about using a similar folder/file structure here?